docs/reference/commandline/stack_deploy.md

```diff
@@ -57,6 +57,8 @@ Creating service vossibility_ghollector
 Creating service vossibility_lookupd
 ```
 
+The Compose file can also be provided as standard input with `--compose-file -`.
```

Comment:
I think this would be better in the help text for the `--compose-file` flag. Something like

```
Path to a Compose file, use "-" to read from stdin
```